src/index.html

```diff
@@ -3,10 +3,45 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
+    <title>Calendar v1.0</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendarBlock
```

Comment:
```suggestion
    <div class="calendar calendar--29 calendar--sun
```
Классы должны быть такого вида